File: kimhyunji/PGS/Level 1/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = -1;
        int sum = 0;
        
        for(int i=0; i<numbers.length; i++) sum+=numbers[i];
        
        if((45-sum)>0) answer = 45-sum;
        
        return answer;
    }
}

File: baekyeonsu/PGS/Lv. 1/없는 숫자 더하기.js
function solution(numbers) {
  const total = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
  const totalSum = total.reduce((a, b) => a + b);
  const sum = numbers.reduce((a, b) => a + b);
  return totalSum - sum;
}
